psi4/default.nix

```diff
@@ -238,6 +262,12 @@ in
       ctest -j $NIX_BUILD_CORES
     '';
 
+    doInstallCheck = true;
+    installCheckPhase = ''
+      export OMP_NUM_THREADS=$NIX_BUILD_CORES
```

Comment:
I would fix the number of core for the test to two. That makes the test more reproducible.
```
export OMP_NUM_THREADS=2
```

Reply:
Good idea, done.